recipes/freeglut/yum_requirements.txt

```diff
@@ -0,0 +1,2 @@
+libXi-devel
+mesa-libGLU-devel
```

Comment:
Actually, we added `mesa-libGL-devel`. We still have to install `mesa-libGLU-devel` for GLU support.